das/aggregator.go

```diff
@@ -65,6 +71,41 @@ func NewServiceDetails(service DataAvailabilityService, pubKey blsSignatures.Pub
 }
 
 func NewAggregator(config AggregatorConfig, services []ServiceDetails) (*Aggregator, error) {
+	if config.L1NodeURL == "none" {
+		return NewAggregatorWithSeqInboxCaller(config, services, nil)
+	}
+	l1client, err := ethclient.Dial(config.L1NodeURL)
+	if err != nil {
+		return nil, err
+	}
+	seqInboxAddress, err := StoreSignerAddressFromString(config.SequencerInboxAddress)
```

Comment:
Probably good to change the function name to something more generic if we want to be able to use it here. Maybe `OptionalAddressFromString`